.eslintrc.yml

```diff
@@ -28,4 +28,8 @@ rules:
     - off
   import/prefer-default-export:
     - off
-
+  import/extensions:
```

Comment:
remove this rule from the backend (since we have it in the root now)
and you need to fix all linter changes in the frontend

Reply:
Which issues from linter do you mean?